test/click_handler_test.dart

```diff
@@ -22,15 +22,17 @@ main() {
     setUp(() {
       router = new MockRouter();
       mockWindow = new MockWindow();
-      mockWindow.location.when(callsTo('get host'))
-          .alwaysReturn(window.location.host);
-      mockWindow.location.when(callsTo('get hash')).alwaysReturn('');
+      when(mockWindow.location.host).thenReturn(window.location.host);
```

Comment:
TODO - consider wrapping these `when` statements into the mocks themselves (if they are consistent across the test suites)